prow/images/buildpack-node-chromium/Dockerfile

```diff
@@ -0,0 +1,66 @@
+# Node buildpack with chromium and its dependencies
+
+FROM eu.gcr.io/kyma-project/prow/test-infra/buildpack-node:v20181119-afd3fbd
+
+RUN curl -sL https://deb.nodesource.com/setup_10.x | bash -
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    procps \
+    gconf-service \
+    libasound2 \
+    libatk1.0-0 \
+    libatk-bridge2.0-0 \
+    libc6 \
+    libcairo2 \
+    libcups2 \
+    libdbus-1-3 \
+    libexpat1 \
+    libfontconfig1 \
+    libgcc1 \
+    libgconf-2-4 \
+    libgdk-pixbuf2.0-0 \
+    libglib2.0-0 \
+    libgtk-3-0 \
+    libnspr4 \
+    libpango-1.0-0 \
+    libpangocairo-1.0-0 \
+    libstdc++6 \
+    libx11-6 \
+    libx11-xcb1 \
+    libxcb1 \
+    libxcomposite1 \
+    libxcursor1 \
+    libxdamage1 \
+    libxext6 \
+    libxfixes3 \
+    libxi6 \
+    libxrandr2 \
+    libxrender1 \
+    libxss1 \
+    libxtst6 \
+    ca-certificates \
+    fonts-liberation \
+    libappindicator1 \
+    libnss3 \
+    lsb-release \
+    xdg-utils \
+    wget \
+    chromium=70.0.3538.110-1~deb9u1 \
+    && apt-get clean
+
+RUN npm install -g eslint-config-react-app@^3.0.4 \
```

Comment:
Will we require updates to those packages? If yes, I would insist on using a mechanism like [this](https://docs.npmjs.com/cli/shrinkwrap) so we can control the package dependencies and modify them outside of the Dockerfile

Reply:
n/a since it's removed from the dockerfile. should we raise an issue in the node-buildpack?